pkg/sdk/testint/tasks_gen_integration_test.go

```diff
@@ -610,36 +701,39 @@ func TestInt_Tasks(t *testing.T) {
 
 	t.Run("resume root tasks within a graph containing more than one root task", func(t *testing.T) {
```

Comment:
Nit: add in the description that it's supposed to fail.

Reply:
Added